lib/registration.pm

```diff
@@ -220,6 +222,32 @@ sub register_addons_cmd {
     }
 }
 
+sub check_registered_system {
+    my ($system) = @_;
+    my $version = get_var('SLE_PRODUCT') . $system;
+    assert_script_run "zypper lr --uri | grep -i $version";
+}
+
+sub check_registered_addons {
+    my ($addonlist) = @_;
+    $addonlist //= get_var('SCC_ADDONS');
+    if (is_sle('>15')) {
```

Comment:
Since we don't know the service pack version of SLE15, to support the migration from SLE12SP5 to SLE15SP1, maybe we can just '=15' and add comments before it.